### ISSUE TITLE
YOLOv1!!!!overfitting!!!!

### ISSUE THREAD

**Issue by njustBShen** [aladdinpersson/Machine-Learning-Collection#108]
When I use the code, the detection accuracy of the training set is very good, but the detection accuracy of the test set is almost equal to 0, which is obviously overfitting.
In fact, the original code is to train an overfitting model, but I have modified some of the code. Why is it still overfitting?

I have made the following modifications：
nn.Dropout(0) -> nn.Dropout(0.5)
WEIGHT_DECAY = 0->WEIGHT_DECAY = 2e-4

This question has bothered me for a long time. I would appreciate it if you could answer it.

**Comment by dokeefemain** [comment]
My take on it is that the model just has poor generalization. Either that or it's not implemented correctly. I experienced the same thing so I decided to try YOLO V3 instead. I mean increasing dropout is the way to go but if it's still overfitting then I would just choose a different model or look at a different YOLO implementation. They also comment on it's issues with generalization in the origional paper "Our model struggles with small objects that appear in groups, such as flocks of birds. Since our model learns to predict bounding boxes from data, it struggles to generalize to objects in new or unusual aspect ratios or configurations."
